repairnator/repairnator-realtime/src/main/java/fr/inria/spirals/repairnator/realtime/ActiveMQPipelineRunner.java

```diff
@@ -0,0 +1,11 @@
+package fr.inria.spirals.repairnator.realtime;
+
+import fr.inria.spirals.repairnator.dockerpool.AbstractPoolManager;
+
```

Comment:
add a one-sentence Javadoc description

Reply:
This might be a duplicate from the first review issue :) , which is already addressed.

Reference to this:
 > add comment explaining that the implementation will come in a future pull-request